ui/src/app/components/help-sidebar.tsx

```diff
@@ -382,14 +385,17 @@ export const HelpSidebar = fp.flow(
       }
     });
 
-    componentDidMount(): void {
+    async componentDidMount() {
+      const {namespace, id} = this.props.workspace;
       this.subscription = participantStore.subscribe(participant => this.setState({participant}));
       this.subscription.add(setSidebarActiveIconStore.subscribe(activeIcon => {
         if (activeIcon !== null) {
           this.setState({activeIcon});
           this.props.setSidebarState(!!activeIcon);
         }
       }));
+      const workspaceCreatorFreeCreditsRemainingResponse = await workspacesApi().getWorkspaceCreatorFreeCreditsRemaining(namespace, id);
```

Comment:
this gets triggered on workspace changes and on moving from the data panel to the analysis panel for the first time, because the help sidebar has different content for that. I will give making this call in the runtime panel another go.